src/ontology/mondo-edit.obo

```diff
@@ -341015,7 +341017,8 @@ xref: MESH:C536932 {source="ORDO:3403/e", source="MONDO:equivalentTo", source="O
 xref: OMIM:107970 {source="MONDO:subClassOf", source="MONDO:relatedTo", source="ORDO:3403/ntbt", source="Orphanet:3403"}
 xref: Orphanet:3403 {source="MONDO:equivalentTo"}
 xref: UMLS:C0265857 {source="ORDO:3403/e", source="MONDO:notFoundInDiseaseSubset", source="Orphanet:3403"}
-is_a: MONDO:0016343 {source="Orphanet:3403"} ! unclassified cardiomyopathy
+is_a: MONDO:0004994 ! cardiomyopathy
```

Comment:
this needs a source annotation.
I recommend giving it the source: Orphanet:3403

Reply:
Done